src/proto/CMakeLists.txt

```diff
@@ -11,13 +11,4 @@ protobuf_generate_cpp(
 )
 
 add_library(mixxx-proto STATIC ${PROTO_SOURCES} ${PROTO_HEADERS})
-
-# Use uppercase PROTOBUF_ prefix for backwards compatibility for CMake < 3.6.0
-target_include_directories(mixxx-proto SYSTEM PUBLIC ${PROTOBUF_INCLUDE_DIR})
-if(PROTOBUF_INCLUDE_DIR AND PROTOBUF_LIBRARIES)
-  target_link_libraries(mixxx-proto PUBLIC ${PROTOBUF_LIBRARIES})
-elseif(PROTOBUF_INCLUDE_DIR AND PROTOBUF_LITE_LIBRARY)
-  target_link_libraries(mixxx-proto PUBLIC ${PROTOBUF_LITE_LIBRARY})
-else()
-  message(FATAL_ERROR "Protobuf or Protobuf-lite libraries are required to compile Mixxx.")
-endif()
+target_link_libraries(mixxx-proto PUBLIC protobuf::libprotobuf-lite)
```

Comment:
This broke local builds for me:
```
CMake Error at CMakeLists.txt:1250 (add_executable):
  Target "mixxx-test" links to target "protobuf::libprotobuf-lite" but the
  target was not found.  Perhaps a find_package() call is missing for an
  IMPORTED target, or an ALIAS target is missing?


CMake Error at CMakeLists.txt:332 (add_library):
  Target "mixxx-lib" links to target "protobuf::libprotobuf-lite" but the
  target was not found.  Perhaps a find_package() call is missing for an
  IMPORTED target, or an ALIAS target is missing?


CMake Error at CMakeLists.txt:1069 (add_executable):
  Target "mixxx" links to target "protobuf::libprotobuf-lite" but the target
  was not found.  Perhaps a find_package() call is missing for an IMPORTED
  target, or an ALIAS target is missing?


CMake Error at src/proto/CMakeLists.txt:13 (add_library):
  Target "mixxx-proto" links to target "protobuf::libprotobuf-lite" but the
  target was not found.  Perhaps a find_package() call is missing for an
  IMPORTED target, or an ALIAS target is missing?


-- Generating done
CMake Generate step failed.  Build files cannot be regenerated correctly.
```

Reply:
This target is defined starting with CMake 3.9. Are you using an older version?

https://cmake.org/cmake/help/v3.9/module/FindProtobuf.html

Reply:
No, I am using CMake 3.18.3.

Reply:
I'm not a protobuf expert, maybe you only have protobuf but not protobuflite? I can make a PR to fall back to the regular protobuf if protobuflite was not found.

Reply:
Yup, that was it.

Reply:
This seems to have broken CMake Windows builds on AppVeyor:
```
CMake Error at C:/Program Files (x86)/CMake/share/cmake-3.16/Modules/FindPackageHandleStandardArgs.cmake:146 (message):
  Could NOT find Protobuf (missing: Protobuf_LIBRARIES) (found version
  "2.6.1")
Call Stack (most recent call first):
  C:/Program Files (x86)/CMake/share/cmake-3.16/Modules/FindPackageHandleStandardArgs.cmake:393 (_FPHSA_FAILURE_MESSAGE)
  C:/Program Files (x86)/CMake/share/cmake-3.16/Modules/FindProtobuf.cmake:624 (FIND_PACKAGE_HANDLE_STANDARD_ARGS)
  src/proto/CMakeLists.txt:2 (find_package)
```